Makefile

```diff
@@ -79,11 +81,15 @@ install:
 testbuild: TESTS := $(shell $(GO) list -tags '$(TAGS)' $(PKG))
 testbuild: GOFLAGS += -c
 testbuild:
+	ROOT="$(DIR)"; \
 	for p in $(TESTS); do \
```

Comment:
I was doing that originally but decided against it because:
* if the last component of two packages is the same, we'll overwrite one
* if you see the output of `util.test`, it's helpful to know that the full package name is `store/engine/util`